components/config/src/config.rs

```diff
@@ -152,6 +153,8 @@ pub struct Config {
     #[serde(skip_serializing, skip_deserializing)] // not a typo, 2 are need
     pub extra_syntax_set: Option<SyntaxSet>,
 
+    pub link_checker: LinkChecker,
```

Comment:
I would like all config options to be in the config component directly.

Reply:
Okay, I can change that. Thanks for the feedback. 

What do you think about the LinkChecker config struct? This way it will be easier to add new options later, like the user agent. Should `url` then move into the struct and `check_url()` convert to a method? Or keep adding arguments while the link checker evolves?